app/client/models/SearchModel.ts

```diff
@@ -12,6 +12,10 @@ import {TableData} from 'app/common/TableData';
 import {BaseFormatter} from 'app/common/ValueFormatter';
 import {Computed, Disposable, Observable} from 'grainjs';
 import debounce = require('lodash/debounce');
+import { makeT } from 'app/client/lib/localization';
+
+// Use 'tt' instade of 't', because 't' is already delared in the upper scope
```

Comment:
I think it's better to use `t` here for consistency with all other code. The lint error about `t` being already declared is in unrelated lines which seem better candidates to rename the variable (e.g. one maps tables, so could replace `t` with `table`; and the other maps pages, so could replace `t` with `page`).

Reply:
oh your right, thanks ! I hadn't quite read linter's error, I changed it